src/components/chips/js/contactChipsController.js

```diff
@@ -18,6 +18,18 @@ MdContactChipsCtrl.prototype.queryContact = function(searchText) {
   return this.contactQuery({'$query': searchText});
 };
 
+MdContactChipsCtrl.prototype.inputKeydown = function(event) {
+  if (!this.separatorKeys || this.separatorKeys.indexOf(event.keyCode) < 0) {
+    return;
+  }
+
+  event.stopPropagation();
+  event.preventDefault();
```

Comment:
A component should only ever use `preventDefault` or `stopPropagation` when when it actually handles the specific key being pressed; this looks like it's doing it for every keypress into the chips component. This would result in blocking standard browser behaviors that people depend on.

Reply:
@jelbourn This should only be called if the specific keypress is mapped to a key code in the `separatorKeys` configuration. Line 22 will immediately return if it's not a matching key so this should not run on every keypress.

Reply:
Ah, I misread that condition.